native/cocos/bindings/manual/jsb_websocket_server.h

```diff
@@ -49,7 +49,8 @@ SE_DECLARE_FUNC(WebSocketServer_Connection_onbinary);
 SE_DECLARE_FUNC(WebSocketServer_Connection_onconnect);
 SE_DECLARE_FUNC(WebSocketServer_Connection_onerror);
 SE_DECLARE_FUNC(WebSocketServer_Connection_onclose);
-SE_DECLARE_FUNC(WebSocketServer_Connection_ondata);
+SE_DECLARE_FUNC(WebSocketServer_Connection_ondata); // deprecated since v3.7, please use onmessage to instead.
```

Comment:
The codes only used by engine. Just remove it if want to deprecated it.

Reply:
I've removed all the useless `SE_DECLARE_FUNC` in the network module.